openhands/runtime/base.py

```diff
@@ -190,6 +193,15 @@ def add_env_vars(self, env_vars: dict[str, str]) -> None:
                 f'Failed to add env vars [{env_vars.keys()}] to environment: {obs.content}'
             )
 
+        # Add to .bashrc for persistence
+        bashrc_cmd = bashrc_cmd.strip()
+        logger.debug(f'Adding env var to .bashrc: {bashrc_cmd}')
```

Comment:
```suggestion
        logger.debug(f'Adding env var to .bashrc: {env_vars.keys()}')
```
Just the keys, at most, they can be secrets